draft-ietf-quic-http.md

```diff
@@ -907,9 +910,9 @@ graceful shutdown of a connection by sending a GOAWAY frame ({{frame-goaway}}).
 The GOAWAY frame contains an identifier that indicates to the receiver the range
 of requests or pushes that were or might be processed in this connection.  The
 server sends a client-initiated bidirectional Stream ID; the client sends a Push
-ID.  Requests or pushes with the indicated identifier or greater are rejected by
-the sender of the GOAWAY.  This identifier MAY be zero if no requests or pushes
-were processed.
+ID ({{server-push}}).  Requests or pushes with the indicated identifier or
```

Comment:
```suggestion
ID; see {{server-push}}.  Requests or pushes with the indicated identifier or
```

Reply:
That reference informs the specific term, not the whole statement; I think the parentheses are more appropriate in this instance.